packages/flutter/lib/src/cupertino/action_sheet.dart

```diff
@@ -0,0 +1,1264 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show ImageFilter;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'scrollbar.dart';
+
+const TextStyle _kActionSheetActionStyle = const TextStyle(
+  fontFamily: '.SF UI Text',
+  inherit: false,
+  fontSize: 20.0,
+  fontWeight: FontWeight.w400,
+  color: CupertinoColors.activeBlue,
+  textBaseline: TextBaseline.alphabetic,
+);
+
+const TextStyle _kActionSheetContentStyle = const TextStyle(
+  fontFamily: '.SF UI Text',
+  inherit: false,
+  fontSize: 13.0,
+  fontWeight: FontWeight.w400,
+  color: _kContentTextColor,
+  textBaseline: TextBaseline.alphabetic,
+);
+
+// _kCupertinoAlertBlurOverlayDecoration is applied to the blurred backdrop to
```

Comment:
nit: Instead of saying `_kCupertinoAlertBlurOverlayDecoration`, just say "This decoration".  That way, if the variable is renamed using a refactoring tool, the name won't be out of sync.